hax/requirements.txt

```diff
@@ -15,6 +15,8 @@ pytest==6.2.4
 pytest-aiohttp==0.3.0
 pytest-mock==3.6.1
 pytest-timeout==1.4.2
+pytest-cov
```

Comment:
I believe these two dependencies should go under `#:runtime-requirements:` as coverage will be gathered at a deployed system.

Reply:
Done